<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>634</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>634</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zenei műfaj meghatározás</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(16, 96, 145);
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QPushButton" name="OpenButton">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>128</y>
           <width>256</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(76, 206, 197);
	border-radius: 20%;
}
QPushButton:hover
{
	background-color: rgb(255,217,0);
}</string>
         </property>
         <property name="text">
          <string>Fájl megnyitása</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ShowWaveformButton">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>208</y>
           <width>120</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 204, 255);
	border-radius: 20%;
}
	QPushButton:hover
{
	background-color: rgb(255,217,0);
}</string>
         </property>
         <property name="text">
          <string>Hullámforma</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ShowSpectrumButton">
         <property name="geometry">
          <rect>
           <x>168</x>
           <y>208</y>
           <width>120</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 204, 255);
	border-radius: 20%;
}
	QPushButton:hover
{
	background-color: rgb(255,217,0);
}</string>
         </property>
         <property name="text">
          <string>Spektrum</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ShowSpectogramButton">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>288</y>
           <width>120</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 204, 255);
	border-radius: 20%;
}
	QPushButton:hover
{
	background-color: rgb(255,217,0);
}</string>
         </property>
         <property name="text">
          <string>Spektogram</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ShowMFCCButton">
         <property name="geometry">
          <rect>
           <x>168</x>
           <y>288</y>
           <width>120</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 204, 255);
	border-radius: 20%;
}
	QPushButton:hover
{
	background-color: rgb(255,217,0);
}</string>
         </property>
         <property name="text">
          <string>MFCC</string>
         </property>
        </widget>
        <widget class="QPushButton" name="PredictButton">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>368</y>
           <width>256</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(76, 206, 197);
	border-radius: 20%;
}
QPushButton:hover
{
	background-color: rgb(255,217,0);
}</string>
         </property>
         <property name="text">
          <string>Műfaj meghatározása</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_predictedgenre">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>448</y>
           <width>256</width>
           <height>64</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(255, 217, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_predictedgenreaccuracy">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>528</y>
           <width>256</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <widget class="QLabel" name="label_title">
            <property name="geometry">
             <rect>
              <x>8</x>
              <y>16</y>
              <width>320</width>
              <height>64</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Zenei műfaj meghatározás</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_coverart">
            <property name="geometry">
             <rect>
              <x>4</x>
              <y>128</y>
              <width>320</width>
              <height>320</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <widget class="QPushButton" name="BackwardMusicButton">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>10</y>
              <width>64</width>
              <height>64</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border-image: url(images/backwardbutton.png); border: 0px;
}
QPushButton:hover
{
	border-image: url(images/backwardbutton_pressed.png); border: 0px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="PlayMusicButton">
            <property name="geometry">
             <rect>
              <x>92</x>
              <y>10</y>
              <width>64</width>
              <height>64</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="StopMusicButton">
            <property name="geometry">
             <rect>
              <x>172</x>
              <y>10</y>
              <width>64</width>
              <height>64</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border-image: url(images/stopbutton.png); border: 0px;
}
QPushButton:hover
{
	border-image: url(images/stopbutton_pressed.png); border: 0px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="ForwardMusicButton">
            <property name="geometry">
             <rect>
              <x>252</x>
              <y>10</y>
              <width>64</width>
              <height>64</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border-image: url(images/forwardbutton.png); border: 0px;
}
QPushButton:hover
{
	border-image: url(images/forwardbutton_pressed.png); border: 0px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <widget class="QLabel" name="label_filename">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>128</y>
           <width>288</width>
           <height>64</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>288</width>
           <height>64</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Fájlnév:</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_songtitle">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>192</y>
           <width>288</width>
           <height>64</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>288</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Zenecím:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_songartist">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>256</y>
           <width>288</width>
           <height>64</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>288</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Előadó:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_songrelease">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>320</y>
           <width>288</width>
           <height>64</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>288</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Kiadás éve:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_songbitrate">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>384</y>
           <width>288</width>
           <height>64</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>288</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Bitráta:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
